draft-ietf-quic-transport.md

```diff
@@ -3374,6 +3367,36 @@ received on all streams, which are used to check for flow control violations. A
 receiver might use a sum of bytes consumed on all contributing streams to
 determine the maximum data limit to be advertised.
 
+
+### Stream 0 Flow Control
+
+The cryptographic handshake stream, Stream 0, is exempt from the
+connection-level data limits established by MAX_DATA.  That is, data sent on
+stream 0 is not counted against the limit expressed by MAX_DATA.  Data on stream
+0 is still subject to stream-level data limits and MAX_STREAM_DATA.
+
+Data sent in the Initial and Retry packets do not count toward stream 0 flow
```

Comment:
I'm still a little bothered by "do not count," because that gives us the on-stream version of #1074.  Maybe say "is allowed to temporarily exceed stream 0's flow control limits"?  Otherwise, it looks like the flow control bound is separated from the stream offset, which isn't what you're going for.